### PR TITLE
initial draft

### PR DESCRIPTION
**Pull request by storopoli**
The first version

### CODE REVIEW

File: docs/reference.md
Comment (jotas6) [inline docs/reference.md:1]: I think that the key points section is missing, or the placeholder text wasn't removed.

image

File: docs/reference.md [hunk 1]
```diff
@@ -12,17 +12,54 @@ This can be either a markdown table or a list.
...
+| Add covariates to a model | `@covariates covar1 covar2 ...` | The `@covariates` block should be used inside a model. Also note that the matching `Population` used in the `fit` with the desired model should also have the same covariates available |
+| Add a dose control parameter to a model | `@dosecontrol begin dcp = (; Cmt=value) end` | The `@dosecontrol` block should be used inside a model. `dcp` is a dose control parameter (`lags`, `bioav`, `rate` or `duration`) and `Cmt` is the compartment name where the DCP effect should be applied and `value` is the value of the effect. You can have multiples `Cmt`s and also multiples `dcp`s. |
+| Parse data with multiple observations into a `Population` | `read_pumas(pkdata; observations=[:obs1, :obs2, ...])` | `observations` is a vector of column names where observation data is stored in the `pkdata` `DataFrame` |
+| Define initial values for compartments in a model | `@init begin Cmt = value end` | The compartment always has a initial value of 0 or the dosing event at time 0 if not specified with `@init` |
```
Comment (jotas6) [inline docs/reference.md:19]:
```suggestion
| Define initial values for compartments in a model | `@init begin Cmt = value end` | The compartment always has an initial value of 0 or the dosing event at time 0 if not specified with `@init` | 
```

**Review by jotas6 — COMMENTED** [review]
Minor suggestions

File: 02-dose_control.jl [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    tvka=0.14,
+    tvcl=1,
+    tvvc=70,
+    tvlag=0.1,
+    tvbio=0.8,
+    Ω=Diagonal([0.05, 0.05, 0.05]),
+    σ_prop=0.015,
+)
+
+fit_lags_bioav = fit(lags_bioav, pop, param_bioav, FOCE())
```
[the remark this answers is not in the document]
Comment (storopoli) [reply 02-dose_control.jl:150]: Yes, thank you!

File: docs/reference.md [hunk 3]
```diff
@@ -1,28 +1,70 @@
...
+title: Reference Sheets for Pumas-AI NLME Covariates, Dose Control Parameters, and PKPD Indirect Response Models Workshop
 ---
 
 [![CC BY-SA 4.0](https://img.shields.io/badge/License-CC%20BY--SA%204.0-lightgrey.svg)](http://creativecommons.org/licenses/by-sa/4.0/)
 
 ## Key Points
 
-This can be either a markdown table or a list.
+- You can parse covariates while reading a `DataFrame` with the `covariates` keyword argument in `read_pumas`.
+- Covariates can be included in a Pumas model with the `@covariates` block and used throughtout.
```
Comment (jotas6) [inline docs/reference.md:10]:
```suggestion
- Covariates can be included in a Pumas model with the `@covariates` block and used throughout.
```

File: docs/reference.md [hunk 4]
```diff
@@ -1,28 +1,70 @@
...
+- Dose control parameters can be defined in the `@dosecontro` in a Pumas model.
```
Comment (jotas6) [inline docs/reference.md:11]:
```suggestion
- Dose control parameters can be defined in the `@dosecontrol` in a Pumas model.
```

**Review by jotas6 — APPROVED** [review]
Some typos

**Pull request merged**